settlement-pipelines/src/stake_accounts.rs

```diff
@@ -23,10 +23,13 @@ use validator_bonds_common::stake_accounts::{
 // TODO: better to be loaded from chain
 pub const STAKE_ACCOUNT_RENT_EXEMPTION: u64 = 2282880;
 
-// prioritize collected stake accounts by:
-// - 1. initialized, non-delegated
-// - 2. deactivating
-// - 3. any non-locked
+// 4bZ6o3eUUNXhKuqjdCnCoPAoLgWiuLYixKaxoa8PpiKk
+pub const MARINADE_LIQUID_STAKER_AUTHORITY: [u8; 32] = [
```

Comment:
Please just use String, then Pubkey::from_str().unwrap()

Reply:
Ok (my thinking was to omit parsing).

Reply:
readability > computer time